styleguide/components/11-base/collections/_collections.scss

```diff
@@ -0,0 +1,40 @@
+.collection {
```

Comment:
What is a collection?

Reply:
It's a class name to use for collections of data, like views in Drupal.

Reply:
@Jeroen005 Placeholder was added to provide collection styling to other elements should it be necessary in the future.